test/uk/gov/hmrc/cardpaymentfrontend/controllers/PaymentCompleteControllerSpec.scala

```diff
@@ -268,6 +269,28 @@ class PaymentCompleteControllerSpec extends ItSpec {
             }
 
           }
+
+          if (origin.toTaxType == vatConsumerToConsumer) {
```

Comment:
Can you put these on line 201, we don't need a conditional here, just have a dedicated test